examples/image_classification/README.md

```diff
@@ -0,0 +1,393 @@
+
+**Note**. Control tasks are tasks with the correct response known in advance. 
+They are used to track the performer's quality of responses. The performer's response is compared to the response you provided. If they match, it means the performer answered correctly.
+
+```java
+var dataset = readTSV("src/main/java/examples/dataset.tsv");
+var goldenDataset = dataset.subList(0, 15);
+var taskDataset = dataset.subList(15, dataset.size());
+```
+
+Function `readTSV` takes file name and returns `List<Map<String, String>>`, where each map keys are colunm names in file 
+and values are values in corresponding row.
+
+Create control tasks. In small pools, control tasks should account for 10–20% of all tasks.
+
+**Tip**: Make sure to include different variations of correct responses in equal amounts.
+
+```java
+var goldenTasks = goldenDataset.stream().map(row -> {
+        var task = new Task(
+        poolId,
+        Map.of("image", row.get("url")),
+        List.of(new KnownSolution(Map.of("result", row.get("label")), 0.0))
+        );
+        task.setInfiniteOverlap(true);
+        return task;
+        }).collect(Collectors.toUnmodifiableList());
+```
+
+Create pool tasks.
+
+```java
+var tasks = taskDataset.stream().map(row -> {
+        var task = new Task(poolId, Map.of("image", row.get("url")));
+        task.setOverlap(3);
+        return task;
+        }).collect(Collectors.toUnmodifiableList());
+```
+
+Upload tasks.
+
+```java
+var taskCreateRequestParameters = new TaskCreateRequestParameters();
+taskCreateRequestParameters.setAllowDefaults(true);
+clientFactory.getTaskClient().createTasks(tasks, taskCreateRequestParameters);
+clientFactory.getTaskClient().createTasks(goldenTasks, taskCreateRequestParameters);
+```
+
+Start the pool.
+
+**Important.** Remember that real Toloka performers will complete the tasks. Double check that everything is correct 
+with your project configuration before you start the pool.
+
+```java
+clientFactory.getPoolClient().openPool(poolId);
+```
+
+## Receiving responses
+
+Wait until the pool is completed.
+
+```java
+Pool pool = clientFactory.getPoolClient().getPool(poolId);
+while (pool.getStatus() != PoolStatus.CLOSED) {
+    try {
+        Thread.sleep(1000 * 60);
+    } catch (InterruptedException ignored) {
+    }
+    pool = clientFactory.getPoolClient().getPool(poolId);
+}
+```
+
+Get responses.
+
+When all the tasks are completed, look at the responses from performers.
+
+Aggregate results using the Dawid-Skene model.
+
+```java
+PoolAggregatedSolutionRequest request = new DawidSkenePoolAggregatedSolutionRequest(poolId, List.of(new DawidSkenePoolAggregatedSolutionRequest.Field("result")));
+AggregatedSolutionOperation result = clientFactory.getAggregatedSolutionClient().aggregateSolutionsByPool(request).waitToComplete();
+var solutionSearchRequest = AggregatedSolutionSearchRequest.make().limit(250).done();
```

Comment:
Lets use `var` instead of typical reference type initialization

two notes:
- do not use `var` with primitives
- variable defined via `var` should have good naming

Reply:
Done